app/src/organisms/ODD/QuickTransferFlow/QuickTransferAdvancedSettings/PushOut.tsx

```diff
@@ -58,58 +68,139 @@ export function PushOut(props: PushOutProps): JSX.Element {
     },
   ]
 
+  const setSaveOrContinueButtonText =
+    pushOutIsEnabled && currentStep < 2
+      ? t('shared:continue')
+      : t('shared:save')
+
   const handleClickBackOrExit = (): void => {
     onBack()
   }
```

Comment:
can we condense this
```suggestion
  const handleClickBackOrExit = onBack
```
Or even just reference `onBack` instead?